src/config.ts

```diff
@@ -12,6 +12,23 @@ const AWSX_PROFILE_PATH = path.join(AWSX_HOME, 'profiles');
 const AWS_CREDENTIALS_PATH = path.join(AWS_HOME, 'credentials');
 const AWS_CONFIG_PATH = path.join(AWS_HOME, 'config');
 
+const configFileCheck = (): void => {
+  try {
+    const hasCredentials = fs.existsSync(AWS_CREDENTIALS_PATH);
+    const hasConfig = fs.existsSync(AWS_CONFIG_PATH);
+
+    !hasCredentials && console.error(`You are missing a required file at: ${AWS_CREDENTIALS_PATH}`);
```

Comment:
Could these error messages tell you more about how to fix the problem? I'm thinking if these files don't exist the user likely doesn't have the aws-sdk installed/configured correctly right? Maybe the message could say something more helpful about that.